include/poncos/poncos.hpp

```diff
@@ -34,11 +34,11 @@ struct sched_configT {
 	std::vector<unsigned char> mems;
 };
 
-constexpr size_t SLOT_SIZE = 8;
+constexpr size_t SLOT_SIZE = 12;
```

Comment:
Yes, you are totally right. I removed the constants `SLOT_SIZE` and `SLOTS` completely.